scripts/release_api_retry.py

```
@@ -232,9 +232,7 @@ def release_by_tag(self, tag: str) -> dict[str, object] | None:
                     return item
             if len(releases) < 100:
                 return None
-        raise ReleaseAPIError(
-            f"GitHub release listing exceeded {MAX_RELEASE_LIST_PAGES} pages while probing {tag!r}"
-        )
+        raise ReleaseAPIError(f"GitHub release listing exceeded {MAX_RELEASE_LIST_PAGES} pages while probing {tag!r}")
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)